doc/specifications/rest/REST-API-V2.rst

```diff
@@ -1454,6 +1455,24 @@ Delete Content Version
     :401: If the user is not authorized to delete this version
     :403: If the version is in state published
 
+Delete Content Version Translation
+``````````````````````
+:Resource: /content/objects/<ID>/versions/<versionNo>/translations/<language_code>
```

Comment:
now this is supposed to happen only on published version, do we really need to put the versionNo in the URI ? I think, we could use `currentversion` to hint that. So the resource URI would be: `/content/objects/<ID>/currentversion/translations/<language_code>`

Reply:
Side note: if we want to follow the philosophy of our REST API, this URI should also be present somewhere in a *previous* response so that we don't have to build it manually when consuming the API. That said, I don't know where/how to do that.

Reply:
> now this is supposed to happen only on published version, do we really need to put the versionNo in the URI ? I think, we could use currentversion to hint that. So the resource URI would be: /content/objects/<ID>/currentversion/translations/<language_code>

Ah, yes, that makes more sense. Fixed Spec in 53da041 (+ additional naming style fix)

> Side note: if we want to follow the philosophy of our REST API, this URI should also be present somewhere in a previous response so that we don't have to build it manually when consuming the API. That said, I don't know where/how to do that.

So that's why you need all these `_href`s in responses... :grinning: 
If I assume correctly that you render Version tab - that will contain this link - from the `/content/objects/<ContentId>/versions` endpoint, 
then, looking at JSON response I could add:

`VersionList.VersionItem.<PublishedVersionNo>.Version.DeleteTranslation.<languageCode>._href`

But I'm not sure yet where to add this (and if it's possible to add it where I suggested) and also if this follows proper naming conventions.
@bdunogier this seems like your area of expertise :) WDYT?